apex/pyprof/prof/utility.py

```diff
@@ -1,4 +1,5 @@
 from functools import reduce
+import pdb
```

Comment:
Is the `pdb` import necessary here or could it be removed?

Reply:
Sorry about that, thanks for the catch, I'll remove it.